recipes/openff-toolkit/meta.yaml

```diff
@@ -10,16 +10,14 @@ source:
   sha256: 38eeab18f5e2c0eeeb06d96ef93397070319a7e55fc9ea61d806f45c43f82cdb
 
 build:
-  noarch: python
   number: 0
-  script: "{{ PYTHON }} -m pip install . -vv --no-deps"
 
 requirements:
   host:
     - python
     - pip
   run:
-    - python >=3.6
```

Comment:
Same here. Otherwise py27 users will be able to install it but not use it (because the toolkit is not py27-compatible right?).

Reply:
I dropped this line because the linter was complaining at my above changes (which seem to be mistaken for other reasons)

Reply:
And yes, the toolkit is 3.6+, no 2.7